src/styles/colors.stories.tsx

```diff
@@ -1,35 +1,35 @@
-// @flow
 import * as React from 'react';
-import * as colors from './variables';
+import * as variables from './variables';
 import mdNotes from './colors.md';
 
-const bdlColors = {};
+const bdlColors: Record<string, Array<any>> = {}; // eslint-disable-line @typescript-eslint/no-explicit-any
+const colors: Record<string, string | Array<string>> = variables;
```

Comment:
I would think TS could infer the type based on the object literal imported from variables.js? Maybe you just need to add an index type?